addon/index.js

```diff
@@ -157,6 +158,15 @@ function initServerEnvironmentPreference() {
   });
 }
 
+function openOnboardingTab() {
+  // TODO: Settings is populated async, so we need a global afterSettingsReady
+  // promise that this function can wait on.
+  tabs.open({
+    url: SERVER_ENVIRONMENTS.production.BASE_URL + '/onboarding',
```

Comment:
FWIW, we basically have an `afterSettingsReady` in [`updatePrefs()`](https://github.com/mozilla/testpilot/blob/dc5f374b956394682c373c9bd890aa152436dd1f/addon/index.js#L77) - that's where we do all the post-environment stuff. Would have to maybe set a global var like `doOpenOnboardingTab` in the startup code and then resolve it in `updatePrefs()` by showing the tab.